examples/Getting_started_with_assistants_api_node.md

```diff
@@ -0,0 +1,331 @@
+# Getting started with the Assistants API using Node.js
+
+The Assistants API makes it easy for developers to create AI agents with features such as knowledge retrieval, memory, function calling, and code execution. It abstracts away complexities that you would normally have to write code for, such as creating text embeddings, interacting with vector databases, storing message history, and converting API responses into function calls.
+
+In this article, you will learn how to use the Assistants API via OpenAI's Node.js SDK. You will build a travel agent that helps users with their vacations by reading their PDFs (e.g., tickets) and performing calculations (e.g., splitting a bill).
+
+If you prefer watching screencasts instead of reading, you can [check out this scrim](https://scrimba.com/scrim/crk9aZCd), which walks through this article. Alternatively, you can enroll in [this free Assistants API crash course.](https://scrimba.com/learn/openaiassistants)
+
+<ScrimPlayer scrimId="cJ9nQeHv" />
+
+## Creating an Assistant
+
+The first thing we need to do is to set up our Assistant. This is done by invoking the `openai.beta.assistants.create()` method and passing in an object where we specify its name, how it should behave, which tools it should have access to, and what model we'll use:
+
+```js
+await openai.beta.assistants.create({
+	instructions: "You are a travel guide. When asked questions, you can use any of the provided tools to get to an answer.",
+	name: "Travel Guide",
+	model: "gpt-4-1106-preview",
+	tools: [{ type: "code_interpreter", type: "retrieval"],
+});
+```
+
+We only need to run the above code once, as OpenAI will save our Assistant on their backend. Once created, it will be listed under the Assistants tab in [the OpenAI developer platform](https://platform.openai.com/assistants), where you can also edit it if you need to.
+
+![Asssitant dashboard open](../images/assistants_overview_assistants_dashboard_open.png)
+
+Let's copy the `id` of our Assistant and store it in a variable, and then use the `openai.beta.assistants.retrieve()` method to get hold of our Assistant in our app.
+
+```js
+const assistantId = "asst_asFsPljnaX10LcwG8tndwK99";
```

Comment:
We could also get this from the above API call, could be beneficial to show that so that users know they don't necessarily need the dashboard.

Reply:
Good point! Just added a paragraph which mentions two alternative ways to get ahold of the id (from the above API call and via `assistants.list()`)